notebooks/00_quick_start/run_notebook_on_aml.ipynb

```diff
@@ -0,0 +1,261 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<i>Copyright (c) Microsoft Corporation. All rights reserved.</i>\n",
+    "\n",
+    "<i>Licensed under the MIT License.</i>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Run Notebook on AML\n",
+    "This noteboook provides an example of how to directly submit notebook to AzureML compute targets. \n",
```

Comment:
This needs more context on what the notebook is doing and why it is beneficial.

Reply:
Added more context